Lib/Src/CUartCom.cpp

```diff
@@ -181,42 +196,25 @@ void CUartCom::endTx()
  */
 void CUartCom::uartRxHandler(UART_HandleTypeDef *p_huart)
 {
-    // Store incoming char
-    static uint8_t len_counter = 0;
+    // Store char in buffer
```

Comment:
You still post superfluous comments that do not actually help with understanding the code.